.eslintrc

```diff
@@ -145,7 +145,10 @@
       },
       "rules": {
         "import/named": "off",
-        "react/prefer-stateless-function": "off"
+        "jest/expect-expect": "off",
+        "react/prefer-stateless-function": "off",
+        "no-use-before-define": "off",
```

Comment:
The base rule has to be disabled before enabling the `@typescript-eslint` one below